src/intl/en/page-upgrades-index.json

```diff
@@ -61,48 +41,50 @@
-  "page-upgrades-index-staking-learn-desc": "The Beacon Chain will bring staking to Ethereum. This means if you have ETH, you can do a public good by securing the network and earn more ETH in the process.",
+  "page-upgrades-index-staking-learn-desc": "The Beacon Chain brought staking to Ethereum. This means if you have ETH, you can do a public good by securing the network and earn more ETH in the process.",
   "page-upgrades-index-staking-step-1": "1. Set up with the launchpad",
   "page-upgrades-index-staking-step-1-btn": "Visit staking launchpad",
   "page-upgrades-index-staking-step-1-desc": "To stake on Ethereum you'll need to use the launchpad - this will walk you through the process.",
   "page-upgrades-index-staking-step-2": "2. Confirm staking address",
   "page-upgrades-index-staking-step-2-btn": "Confirm deposit contract address",
   "page-upgrades-index-staking-step-2-desc": "Before you stake your ETH, be sure to check you've got the right address. You must have gone through the launchpad before doing this.",
   "page-upgrades-index-staking-sustainability": "More sustainable",
-  "page-upgrades-just-docking": "More on The Merge",
   "page-upgrades-meta-desc": "An overview of the Ethereum upgrades and the vision they hope to make a reality.",
   "page-upgrades-meta-title": "Ethereum upgrades (formerly 'Eth2')",
-  "page-upgrades-miners": "More on miners",
-  "page-upgrades-more-on-upgrades": "More on the Ethereum upgrades",
   "page-upgrades-proof-of-stake": "proof-of-stake",
   "page-upgrades-proof-of-work": "proof-of-work",
   "page-upgrades-proof-stake-link": "More on proof of stake",
   "page-upgrades-question-1-title": "When will the upgrades ship?",
   "page-upgrades-question-1-desc": "Ethereum is being upgraded progressively; the upgrades are distinct with different ship dates.",
   "page-upgrades-question-2-title": "Is the Beacon Chain a separate blockchain?",
-  "page-upgrades-question-2-desc": "It's not accurate to think of these upgrades as a separate blockchain.",
-  "page-upgrades-question-3-answer-2": "The Merge and shard chain upgrades may impact dapp developers. But the specifications have not been finalized yet, so there's no immediate action required.",
+  "page-upgrades-question-2-desc": "It's not accurate to think of this upgrade as a separate blockchain.",
+  "page-upgrades-question-3-answer-2a": "The Merge specifications have been finalized, and is designed to have minimal impact on dapp developers.",
+  "page-upgrades-question-3-answer-2a-link": "The Merge and dapp developers",
+  "page-upgrades-question-3-answer-2b": "Sharding plans are still being developed, but will be designed with layer 2 rollups in mind.",
+  "page-upgrades-layer-2-rollups": "More on layer 2 rollups",
   "page-upgrades-question-3-answer-3": "Talk to the Ethereum research and development team over at ethresear.ch.",
   "page-upgrades-question-3-answer-3-link": "Visit ethresear.ch",
```

Comment:
Should we remove this? Could argue it detracts from the purpose of this card...

Reply:
Remove what?

Reply:
Oh sorry, GitHub wasn't showing which lines in the other view... Yeah, remnant probably not needed

Reply:
It's also in other cards 👍